TouchEffectSample.Android/Resources/Resource.designer.cs

```diff
@@ -107,8 +107,8 @@ public static void UpdateIdValues()
 			global::TouchEffect.Droid.Resource.Attribute.behavior_overlapTop = global::TouchEffectSample.Droid.Resource.Attribute.behavior_overlapTop;
 			global::TouchEffect.Droid.Resource.Attribute.behavior_peekHeight = global::TouchEffectSample.Droid.Resource.Attribute.behavior_peekHeight;
 			global::TouchEffect.Droid.Resource.Attribute.behavior_skipCollapsed = global::TouchEffectSample.Droid.Resource.Attribute.behavior_skipCollapsed;
-			global::TouchEffect.Droid.Resource.Attribute.borderWidth = global::TouchEffectSample.Droid.Resource.Attribute.borderWidth;
 			global::TouchEffect.Droid.Resource.Attribute.borderlessButtonStyle = global::TouchEffectSample.Droid.Resource.Attribute.borderlessButtonStyle;
+			global::TouchEffect.Droid.Resource.Attribute.borderWidth = global::TouchEffectSample.Droid.Resource.Attribute.borderWidth;
```

Comment:
the file should be discarded

Reply:
Isn't resource.designer.cs automatically generated?